docker/docker-chaos/metrictank.ini

```diff
@@ -180,6 +180,10 @@ create-cf = true
 schemas-file = /etc/metrictank/storage-schemas.conf
 # path to storage-aggregation.conf file
 aggregations-file = /etc/metrictank/storage-aggregation.conf
+# enables/disables the enforcement of the future tolerance limitation
+enforce-future-tolerance = true
```

Comment:
i feel that we dont need two config settings here.  Can we just disable enforcement if future-tolerance-ratio=0

Reply:
isn't it a realistic use case that somebody wants to reject all future data? if 0 disables the limitation, then this wouldn't be possibles.

Reply:
If we want to disable this feature via a ratio value, then we should probably rather use something like `100`. theoretically, it would currently also be possible to use a higher value like `200`, in which case we'd accept future datapoints up to `2 * <raw ttl>` into the future, but realistically it seems unlikely that somebody wants that.

Reply:
given the desire to be able to deploy this feature in a monitoring only way first (ie just increment a metric counter when the future tolerance is exceeded but dont drop datapoints), the two options makes sense.